pkgs/development/compilers/ghc/respect-ar-path.patch

```diff
@@ -0,0 +1,25 @@
+diff -urd a/aclocal.m4 b/aclocal.m4
```

Comment:
Generally we don't carry patches around in nixpkgs if we can avoid it.  Can you pull this from upstream with `fetchpatch`?

Reply:
Yes, it would be good to know what the origin of that patch is and an easy way to document that is to just `fetchPatch` it from the origin.

Reply:
We can't use the upstream patch in this case. The upstream patch only fixes `aclocal.m4`, but the source tarballs we fetch from upstream contains auto-generated `configure` script which also needs to be patched, so the in-tree patch needs to patch `configure` as well.